pkg/tests/apis/openapi_snapshots/notifications.alerting.grafana.app-v0alpha1.json

```diff
@@ -3434,6 +3433,11 @@
           }
         },
         "x-kubernetes-group-version-kind": [
+          {
+            "group": "notifications.alerting.grafana.app",
+            "kind": "Receiver",
+            "version": "__internal"
+          },
```

Comment:
it doesn't seem like this should be showing up here 🤔

Reply:
Other apps have it to. Perhaps, we need to fix it in the framework